src/info/langs/language.rs

```diff
@@ -2,14 +2,38 @@ use colored::Color;
 use serde::Serialize;
 use std::env;
 use strum::{EnumIter, EnumString, IntoStaticStr};
+use OptionalColor::*;
 
 pub struct Colors {
-    basic_colors: Vec<Color>,
-    true_colors: Option<Vec<Color>>,
+    basic_colors: Vec<OptionalColor>,
+    true_colors: Option<Vec<OptionalColor>>,
 }
 
+macro_rules! define_optional_color {
+    ($( $color:ident ),*) => {
+        #[derive(Debug)]
+        pub enum OptionalColor {
+            $( $color, )*
+            TrueColor{ r: u8, g: u8, b: u8 },
+            Fg,
```

Comment:
We should maybe add a comment to explain what is `Fg` for.

Reply:
Yeah, that would work (and be more similar to neofetch). Perhaps `White` and `Fg` can be effective aliases of each other?